tests/integration/pre_run_script.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+# Pre-run script for integration test operator-workflows action.
+# https://github.com/canonical/operator-workflows/blob/main/.github/workflows/integration_test.yaml
+
+# Jenkins machine charm is deployed on lxd and Jenkins agent charm is deployed on microk8s.
```

Comment:
Is this comment accurate?

Reply:
Woops, it's the other way round, thanks!